### PR TITLE
fix inflated read counts from multimapping

### LINKED ISSUE #80: Enabling atac.multimapping inflates read counts in QC reports
When I run the pipeline with atac.multimapping > 1, the following read count statistics are inflated in the QC report:

- flaqstat_qc: total
- flagstat_qc: mapped
- ataqc: read count from sequencer
- ataqc: read count successfully aligned
- ataqc: read count after filtering for mapping quality
- ataqc: read count after removing duplicate reads
- ataqc: read count after removing mitochondrial reads (final read count) - this does correspond to the number of reads in the filtered BAM file, but it misrepresents the number of reads removed due to MT

Can you please adjust the code to output statistics for just the primary reads at all of these steps? The inflated counts are very difficult to interpret.

### PR DESCRIPTION
**Pull request by vervacity**
makes sure to filter out nonprimary alignments so that `samtools -c` will properly count reads vs alignments. resolves #80

### CODE REVIEW

File: src/encode_ataqc.py [hunk 1]
```diff
@@ -200,11 +203,11 @@ def ataqc():
         flagstat_map = parse_flagstat_qc(args.flagstat_log)
         with open(args.flagstat_log,'r') as fp:
             flagstat = fp.read()
-        mapped_count = flagstat_map['mapped']
+        #mapped_count = flagstat_map['mapped']
```
Comment (strattan) [inline src/encode_ataqc.py]: It's usually best not to leave commented-out lines of code.  Here you're assigning mapped_count at a new place (above).  There's no need to leave this comment here to indicate that this is where it was assigned before.  The git history is adequate for that.

File: src/encode_ataqc.py [hunk 2]
```diff
@@ -200,11 +203,11 @@ def ataqc():
...
     else:
         flagstat = None
-        mapped_count = None
-
+        #mapped_count = None
```
Comment (strattan) [inline src/encode_ataqc.py]: Same as above.

File: src/encode_ataqc.py (hunk 1 above)
Comment (vervacity) [reply src/encode_ataqc.py]: sounds good, have removed in latest commit!

File: src/encode_ataqc.py (hunk 2 above)
Comment (vervacity) [reply src/encode_ataqc.py]: have resolved in latest commit!

**Comment by leepc12** [comment]
@vervacity Please pull first and then make modification to the branch. CI test failed because the following was re-inserted to the code.

https://github.com/ENCODE-DCC/atac-seq-pipeline/pull/118/commits/2f7857b2604c662254ec8443fb9ec8ce76a3492f#diff-09d56ebdb7bfb4c9e9be1f7f19919e27R205

I will make a fix commit for this.

**Pull request merged**